src/util/fs.js

```diff
@@ -721,9 +721,9 @@ export async function symlink(src: string, dest: string): Promise<void> {
       // use relative paths otherwise which will be retained if the directory is moved
       let relative;
       if (await exists(src)) {
-        relative = path.relative(fs.realpathSync(path.dirname(dest)), fs.realpathSync(src));
+        relative = path.relative(fs.realpathSync(path.dirname(dest)), fs.realpathSync(src)) || '.';
       } else {
-        relative = path.relative(path.dirname(dest), src);
+        relative = path.relative(path.dirname(dest), src) || '.';
       }
       await fsSymlink(relative, dest);
```

Comment:
Do `await fsSymlink(relative || '.', dest);` instead?

Also please add a comment above why we default to `'.'` (basically, the summary of this PR as a comment :))